deploy/complete/helm-chart/mushop/charts/assets/templates/destroy-job.yaml

```diff
@@ -0,0 +1,53 @@
+{{- if .Values.global.osb.objectstorage }}
+{{- $credentialSecret := .Values.ociAuthSecret | default .Values.global.ociAuthSecret }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "assets.fullname" . }}-destroy
+  labels: {{ include "assets.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": pre-delete
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+spec:
+  ttlSecondsAfterFinished: 60
```

Comment:
The ttl is working for this job and correctly removing/recycling the pod after finished?

Reply:
It is cleaning, yes.